change/@azure-msal-node-extensions-0bdf3389-5c5a-4ab3-8520-ff2adb9fb815.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "prerelease",
```

Needs to be prereleases which extensions is still in alpha/beta

---

My bad, keeps slipping my mind.